orga/translations/en.json

```diff
@@ -289,8 +289,10 @@
       "no": "No",
       "purpose": {
         "assessment": "Assess participants",
+        "assessment-info": "An assessment campaign tests participants to on specific topics.",
```

Comment:
```suggestion
        "assessment-info": "An assessment campaign tests participants on specific topics.",
```
non ?

Reply:
Dans la trad qui a été donnée c'est bien écrit `to on`. je vais demander confirmation

Reply:
je viens de demander et tu as raison